source-kafka/src/kafka.rs

```diff
@@ -312,7 +312,7 @@ fn parse_message<'m>(msg: &'m BorrowedMessage<'m>) -> Result<serde_json::Value,
     })
 }
 
-static KAFKA_INTERNAL_TOPICS: [&str; 1] = ["__consumer_offsets"];
+static KAFKA_INTERNAL_TOPICS: [&str; 2] = ["__consumer_offsets", "__amazon_msk_canary"];
```

Comment:
just for future reference, you can use a slice (`&[&str]`), which doesn't require an explicit length